src/github.com/matrix-org/go-neb/testutils/testutils.go

```diff
@@ -0,0 +1,15 @@
+package testutils
+
+import (
+	"net/http"
+)
+
+// MockTransport implements RoundTripper
+type MockTransport struct {
+	RT func(*http.Request) (*http.Response, error)
```

Comment:
If you are going to split this out to a library then you should probably add some comments to explain what's going on.